database.sql

```diff
@@ -254,7 +304,7 @@ CREATE TABLE `users` (
+INSERT INTO `users` VALUES (32,'admin','$2a$10$9DSzAbB5BXPTNj22FWgJO.ohefxLelKS1sGzq5AQzRNWoNYpmDtHu','Alice','Nguyen','admin@gmail.com','LOCAL',2,'http://res.cloudinary.com/dwnb2nqcu/image/upload/v1664125885/l6glw6hhd5tooqh0uhhh.jpg'),(33,'boss','$2a$10$ERIORYKMjnrX8KEmabKVxuTmiK3k44oAP7saio8XAmrEKka7x49oi','Boss','Boss','boss@gmail.com','LOCAL',1,NULL),(49,'alice2529','$2a$10$ufxvUV.tZcLzfTjwk6iUP.vzckWdHKuKRK3YzJOlT.0rlA7GWIIku','Alice','Nguyen','user2@gmail.com','LOCAL',5,NULL),(52,'test222','$2a$10$.hm9mBlEasU8ZiLjRAOueuGlFCwwb5qTTOi.klJmoc0f8G57GRB26','Hello','Test','test22@gmail.com','LOCAL',4,NULL),(57,'alicetest@gmail.com','$2a$10$xbYv8bM201Sr86.HvMPDY.TxG7c/azN9sLWo4FfJLY.VYHth4pQ6e','Alice','Test','alicetest@gmail.com','OKTA',2,NULL),(58,'huong.nguyen@hcl.com','$2a$10$JxC.OQFodzNjGkxgKkZeo.El0Jq.UAiUTVQvJoOar4up8L/c9ohyy','Alice','Huong Nguyen','huong.nguyen@hcl.com','LOCAL',1,NULL),(60,'testnewuser','$2a$10$MW9al5e9sb9iwio9gZ2YjOhZMgnLyps8mKrbywFq7r8qkdBOTQeOy','Alice','Hello','test@gmail.com','LOCAL',2,NULL),(64,'alicenguyen2901','$2a$10$CAS6iCORONH12sWOyzeX9eolj.vv63lG18sZhgu/LCcSMo2En5PDO','Alice','Nguyen','user@gmail.com','LOCAL',11,'http://res.cloudinary.com/dwnb2nqcu/image/upload/v1663766774/jmtnzprn24nlhncewnvq.jpg'),(82,'addresstest','$2a$10$TMS02fUw2SzVxCdmaQKD9.ZJa71/BxTx2jn7TyTu1Y3MYIwnFq.1W','Test','Test','addresstest@gmail.com','LOCAL',18,'http://res.cloudinary.com/dwnb2nqcu/image/upload/v1663358361/rwwvxanepovsyumllh37.jpg'),(83,'alicehnguyen','$2a$10$Eg8KnzmvPzAVvR5PRXDCf.qiYW3Yj2eHPfIqMY.wYN/IPNrXeNedW','Alice','N','alicehnguyen@gmail.com','LOCAL',NULL,NULL),(84,'taylor1234','$2a$10$9DSzAbB5BXPTNj22FWgJO.ohefxLelKS1sGzq5AQzRNWoNYpmDtHu','Taylor','Swift','taylorw@gmail.com','LOCAL',NULL,NULL),(86,'taylor12346','$2a$10$WgBvuQKSGJPZdqpLnS2ehu4oaS4395BRlOhtHDab.GsnBWCmXRIrm','Taylor','Swift','taylorw123@gmail.com','LOCAL',NULL,NULL);
```

Comment:
Did you change the admin password here?

Reply:
it's still 12345 just different encryption